### PR TITLE
Ext employees

### PR DESCRIPTION
**Pull request by whites11**
Towards: https://github.com/giantswarm/giantswarm/issues/11732

This PR uses GS users' data from the `employees` repository for the azure platform

### CODE REVIEW

File: platforms/azure/giantnetes/variables.tf [hunk 1]
```diff
@@ -1,3 +1,14 @@
+variable "github_token" {
+  type        = string
+  description = "Your personal GITHUB token, used to get access to the private repository 'employees' to get the list of users."
```
Comment (calvix) [inline platforms/azure/giantnetes/variables.tf:3]: how will this work for CI?

Comment (whites11) [reply platforms/azure/giantnetes/variables.tf:3]: I guess we need to add the variable to conveyor jobs.
Will test that scenario before I merge.

**Comment by calvix** [comment]
this repository was mostly considered public since it was meant to be publicly usable if anyone needs to, but lately, its got more and more focus towards our  platform that its almost impossible to sue outside of GS so I wonder if we should put that info somewhere or maybe even make the repo private

**Comment by whites11** [comment]
> this repository was mostly considered public since it was meant to be publicly usable if anyone needs to, but lately, its got more and more focus towards our platform that its almost impossible to sue outside of GS so I wonder if we should put that info somewhere or maybe even make the repo private

Dropped a note in sig infra. Let's discuss this during next sync

File: platforms/azure/giantnetes/variables.tf (hunk 1 above)
Comment (whites11) [reply platforms/azure/giantnetes/variables.tf:3]: Addressed with https://github.com/giantswarm/conveyor/pull/87

**Comment by whites11** [comment]
Manual tests worked very well on both AWS and Azure.
Trying to make the e2e tests green before I merge

**Comment by whites11** [comment]
e2e tests are broken, but not because of my change.
I can't afford to work on those now, will drop a note in sig infra board and merge this branch.

**Pull request merged**